blosc/schunk.c

```diff
@@ -34,6 +31,12 @@
 
 #endif  /* _WIN32 */
 
+#if defined(_WIN32)
```

Comment:
This includes MSVC, which currently works.  I think a better solution would be to add another condition so that the `mkdir` redefinition only happens on MinGW.